build.cake

```diff
@@ -1,8 +1,8 @@
-#addin nuget:?package=Cake.FileHelpers&version=3.2.1
```

Comment:
This whole cakebuild script change is nice, but it is not related to semver. Therefore it should be a separate PR.